### ISSUE TITLE
Error installing alacritty

### ISSUE THREAD

**Issue by JokerQyou** [cema-sp/homebrew-tap#1]
```zsh
$ brew install --HEAD cema-sp/tap/alacritty
Updating Homebrew...
==> Installing alacritty from cema-sp/tap
==> Cloning https://github.com/jwilm/alacritty.git
Updating /Users/joker/Library/Caches/Homebrew/alacritty--git
==> Checking out branch master
Error: Could not find 'cargo' installation dir neither in CARGO_HOME nor in HOME
```

(I've previously tapped your repo with `brew tap cema-sp/homebrew-tap`)

**Comment by OJFord** [comment]
Yep, I've got the same issue - the check for `$CARGO_HOME` is broken.

Like me, you're probably not using your home directory; my `$CARGO_HOME = $HOME/.config/cargo` for example.

There should not be a further `.cargo` directory inside here, which is what the formula checks for before failing here.

It works if you have `$CARGO_HOME = $HOME/.cargo` currently only because it falls back on using `$HOME`, and looking for `.cargo` inside - which is where your `$CARGO_HOME` was pointing anyway.

TL;DR - it needs to check first for `$CARGO_HOME`; then for `$HOME/.cargo`

cc @cema-sp

**Comment by cema-sp** [comment]
@JokerQyou @OJFord thank you for reporting and investigating issue, please retry to install **alacritty**, https://github.com/cema-sp/homebrew-tap/commit/5d0688fda06a536189363a52138776ecd906d702 should fix it.

**Comment by OJFord** [comment]
Cheers @cema-sp, that's got it.

**Comment by devth** [comment]
quickfix:

```bash
  brew install rustup-init
  rustup-init -y
```

**Comment by specious** [comment]
@cema-sp, I ran into the same error because I didn't have the rust toolchain installed:

```
$ brew install --HEAD cema-sp/homebrew-tap/alacritty
Error: Could not find 'cargo' installation dir neither in CARGO_HOME nor in HOME
```
@devth's suggestion worked for me.  Maybe it would make sense to suggest installing the rust toolchain when exiting due to the missing `cargo` path:

https://github.com/cema-sp/homebrew-tap/blob/5d0688fda06a536189363a52138776ecd906d702/Formula/alacritty.rb#L15-L17
